src/Compilers/Core/Portable/MetadataReference/ModuleMetadata.cs

```diff
@@ -175,7 +179,9 @@ public static ModuleMetadata CreateFromStream(Stream peStream, PEStreamOptions o
             }
 
             // ownership of the stream is passed on PEReader:
+#pragma warning disable CA2000 // Dispose objects before losing scope - ModuleMetadata has dispose ownership
             return new ModuleMetadata(new PEReader(peStream, options));
```

Comment:
>return new ModuleMetadata(new PEReader(peStream, options)); [](start = 12, length = 59)

Why is this tagged. The PEReader disposes the Stream and ModuleMetadata disposes PEReader.

Reply:
This needs https://github.com/dotnet/roslyn-analyzers/issues/1617 to precisely determine dispose ownership transfer without heuristics that can mask true leaks.

Reply:
Should the comment link to the issue?

Reply:
Yes, I plan to add a commit linking all contentious suppressions to tracking issue(s).